slices_test.go

```diff
@@ -216,3 +216,24 @@ func TestReverse(t *testing.T) {
 		}
 	})
 }
+
+func TestUnique(t *testing.T) {
+	t.Run("make unique", func(t *testing.T) {
```

Comment:
Can we add a test case for `nil` or empty slice? Just to be on the safe side for future expansions.